incubator/scylladb/templates/scylla-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scylla
```

Comment:
Please use the fullname template here and other instances `name: {{ template "scylladb.fullname" . }}`